<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.8.0 -->
<title>Research | Salwan Butrus</title>
<meta name="generator" content="Jekyll v4.2.2" />
<meta property="og:title" content="Research" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="© 2022 Salwan Butrus" />
<meta property="og:description" content="© 2022 Salwan Butrus" />
<link rel="canonical" href="http://localhost:4000/research/" />
<meta property="og:url" content="http://localhost:4000/research/" />
<meta property="og:site_name" content="Salwan Butrus" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Research" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"WebPage","description":"© 2022 Salwan Butrus","headline":"Research","url":"http://localhost:4000/research/"}</script>
<!-- End Jekyll SEO tag -->
<link rel="stylesheet" href="/assets/main.css"><link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="Salwan Butrus" /></head>
<body><header class="site-header" role="banner">

  <div class="wrapper-header" style=" float:center; padding-right:20%;padding-left:24.5%"><a class="site-title" rel="author" href="/">Salwan Butrus</a><nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.032C17.335,0,18,0.665,18,1.484L18,1.484z M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.032C17.335,6.031,18,6.696,18,7.516L18,7.516z M18,13.516C18,14.335,17.335,15,16.516,15H1.484 C0.665,15,0,14.335,0,13.516l0,0c0-0.82,0.665-1.483,1.484-1.483h15.032C17.335,12.031,18,12.695,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger"><a class="page-link" href="/">About Me</a><a class="page-link" href="/research/">Research</a><a class="page-link" href="/teaching_outreach/">Teaching &amp; Outreach</a><a class="page-link" href="/collaborators/">Collaborators</a><a class="page-link" href="/hobbies/">Hobbies</a></div>
      </nav></div>
</header>
<main class="page-content" aria-label="Content">
      <div class="wrapper">
        <h1 id="graduate-research"><strong>Graduate Research</strong></h1>

<h3 id="i-deciphering-the-molecular-maturation-of-mouse-retinal-ganglion-cells"><strong>I. Deciphering the molecular maturation of mouse retinal ganglion cells</strong></h3>

<p>The development and connectivity of retinal ganglion cells (RGCs), the retina’s sole output neurons, are patterned by activity-independent transcriptional programs and activity-dependent remodeling. To inventory the molecular correlates of these influences, I applied high throughput single-cell RNA sequencing (scRNA-seq) to mouse RGCs at six embryonic and postnatal ages. I identified temporally regulated modules of genes that correlate with key phases of RGC development, ranging from differentiation and axon guidance to synaptic recognition and refinement. Next, I used these results as a foundation to analyze the transcriptomes of RGCs in mice lacking visual experience due either to dark rearing from birth or to genetic mutations that ablate bipolar or photoreceptor cells. 98.5% of visually deprived (VD) RGCs could be unequivocally assigned to a single RGC type based on their transcriptional profiles, suggesting that visual activity is dispensable for diversification of multipotential precursors into discrete types. However, visual deprivation significantly reduced the transcriptomic distinctions between RGCs, implying that activity influences RGC maturation or maintenance. These results provide a resource for mechanistic analyses of RGC differentiation and maturation, and for investigating the role of activity in these processes.</p>

<p><img src="/images/RGC_VD.png" style="float:center;width:100%;height:auto;" /></p>

<ul>
  <li>*Whitney, I., <strong>*Butrus, S.</strong>, Sanes, J.R., Shekhar, K. <ins>Vision-dependent and -independent molecular maturation of mouse retinal ganglion cells</ins>. <em>Under review at European Journal of Neuroscience.</em></li>
</ul>

<h3 id="ii-characterization-of-visual-cortex-plasticity-at-cell-type-resolution"><strong>II. Characterization of visual cortex plasticity at cell type resolution</strong></h3>

<p>I began my PhD with an interest in combining molecular biology, experimental neuroscience, and machine learning to study visual cortex development at the resolution of its hundreds of cell types. I led a collaboration between our group and Profs. Larry Zipursky and Josh Trachtenberg’s groups at UCLA to study the development of visual cortex cell types as a function of sensory input. The role of postnatal experience in sculpting cortical circuitry, while long appreciated, is poorly understood at the level of cell types. We combined single-nucleus (sn) transcriptomics, statistical inference, sensory perturbations, genetics, and in vivo functional imaging to assemble a developmental transcriptomic atlas of postnatal mouse V1. Using this as a foundation, we discovered that: (1) vision is required for the establishment and maintenance of L2/3 glutamatergic types, but not other cell types in V1; (2) L2/3 glutamatergic cell types are organized as sublayers in V1 and form a transcriptomic continuum through the graded expression of ~200 genes; and (3) among these genes, Igsf9b, a vision-regulated cell adhesion molecule, is required in a graded fashion for the functional maturation of L2/3 glutamatergic neurons. In addition to challenging the prevailing hypothesis that genetically-hardwired mechanisms establish a static blueprint of cell types, our study establishes a framework for future investigations of how experience regulates neuronal organization in the brain.</p>

<p><img src="/images/GraphicalAbstract_v7.png" style="float:center;width:100%;height:auto;" /></p>

<ul>
  <li>*Cheng, S., <strong>*Butrus, S.</strong>, *Tan, L., Sagireddy, S., Trachtenberg, J.T., Shekhar, K., Zipursky, L. <ins>Vision-dependent specification of cell types and function in the developing cortex</ins>. <em>Cell</em> (2022) 185, 2: pp. 311-327.
    <ul>
      <li>Highlighted in <a href="https://www.nature.com/articles/d41586-022-00463-2"><em>Nature</em></a> by Puiggros and Jabaudon (2022).</li>
      <li>Highlighted in <a href="https://chemistry.berkeley.edu/news/research-shines-light-development-visual-cortex-during-critical-period-after-birth"><em>UC Berkeley, College of Chemistry Press Release</em></a> (2022).</li>
      <li>Highlighted in <a href="https://www.eurekalert.org/news-releases/941779"><em>EurekAlert</em></a> (2022).</li>
    </ul>
  </li>
  <li><strong>Butrus, S.</strong>, Shekhar, K. <ins>Vision mediates molecular patterning in the developing visual cortex.</ins> <em>Society for Neuroscience Meeting</em> (2021).</li>
  <li><strong>Butrus, S.</strong>, Shekhar, K. <ins>Vision mediates molecular patterning in the developing visual cortex.</ins> <em>UC Berkeley Center for Computational Biology Retreat</em> (2021).</li>
</ul>

<h3 id="iii-python-package-for-studying-neuroprotection-and-axon-regeneration-using-single-cell-rna-sequencing"><strong>III. Python package for studying neuroprotection and axon regeneration using single-cell RNA sequencing</strong></h3>

<p>A major hallmark of the mammalian central nervous system is that certain populations of neurons exhibit far greater vulnerability to insults than others. The mechanisms underlying this selective neuronal vulnerability in the context of acute (e.g. traumatic injury) or chronic (e.g. neurodegenerative) disease have been difficult to dissect, but recent advances in single-cell RNA-seq (scRNA-seq) make it possible to compare patterns of gene expression among neuronal types that differ in vulnerability. As a postdoctoral fellow, my advisor Prof. Karthik Shekhar explored this strategy in the context of axotomy by analyzing the responses of mouse retinal ganglion cells (RGCs) to optic nerve crush (ONC), a well-studied model of traumatic injury. RGCs are a diverse class of projection neurons, with their diversity in mice comprising &gt;40 discrete types, each with distinct morphological, physiological and molecular features. Using high-throughput scRNA-seq, he derived an atlas of 46 molecularly distinct RGC types. Following ONC, he found that ~85% of RGCs die within two weeks, and those that survive do not regenerate axons. To determine if specific types are lost at the same or different rates, he profiled RGCs using scRNA-seq from injured retinas at 0, 0.5, 1, 2, 4, 7, and 14 days post crush (dpc) and found cell type-specific axonal injury and regeneration patterns.</p>

<p><img src="/images/iGraphBoost.png" style="float:center;width:100%;height:auto;" /></p>

<p>To generalize the computational methods used in this project to any study of neuronal vulnerability and make them accessible to researchers without an extensive background in machine learning, I developed a <a href="https://github.com/shekharlab/mimb_onc_rgc">package</a> in the programming language Python and wrote a step-by-step protocol describing its implementation. I described the computational framework used to analyze transcriptomic profiles of injured RGCs over the aforementioned time course, aiming to assess type-specific differences in survival, and identify gene signatures that correlate with, and may underlie, these differences. Following injury, extensive gene expression changes in RGCs make the inference of type identity challenging. To address this, I used an iterative supervised classification approach that leverages data from RGCs collected at the intermediate time points. In this approach, transcriptomic signatures of RGC types are successively redefined at each time to map the cells at the next time point. The underlying method, called iterative-GraphBoost (iGraphBoost), combines a two-step procedure involving gradient boosted trees (Step 1) and a graph-based voting scheme (Step 2). This dramatically increases cell type mapping efficiency at later time points and allows us to deconvolve gradual injury-related “state” changes from intrinsic type-specific gene expression programs.</p>

<ul>
  <li><strong>Butrus, S.</strong>, Sagireddy, S., Shekhar, K. <ins>Defining selective neuronal resilience and identifying targets of neuroprotection and axon regeneration using single-cell RNA sequencing – computational approaches</ins>. <em>Methods in Molecular Biology</em> (2022). <em>In press</em>.</li>
</ul>

<h1 id="undergraduate-research"><strong>Undergraduate Research</strong></h1>

<h3 id="i-development-and-characterization-of-gold-nanoparticles-for-plant-genetic-engineering"><strong>I. Development and characterization of gold nanoparticles for plant genetic engineering</strong></h3>

<p>Plant genetic engineering is poised to strengthen the agricultural, pharmaceutical, and energy industries in the face of climate change and global population growth. However, traditional biomolecule delivery methods to plants are limited by low efficiency, narrow host range, toxicity, and limited practical applicability due to the transport challenge posed by the plant cell wall. I was intrigued by early work in using carbon nanotubes to transform mature plants in a non-toxic, passive, and species-independent manner that requires no transgene integration. I sought to gauge the promise of gold nanoparticles (AuNPs) as plant genetic engineering tools. By systematically tuning key physicochemical properties of the NPs to optimize bio-cargo loading efficiency, I developed the first two AuNP-based methods for delivering small interfering RNA and plasmid DNA to intact leaves for gene silencing and expression, respectively.</p>

<p>The results sparked fundamental and applied questions involving the use of AuNPs for the delivery of biomolecules to plants and elucidation of the currently elusive structure-function relationships of nanomaterials in plants. The work also elucidated the promise nanomaterials hold in harnessing plants for healthy crops, cheaper medicines, and more efficient biofuels. In addition, the project I conceived through my NSF GRFP <a href="/files/NSFResearchSalwanButrus.pdf">proposal</a> ultimately resulted in a <em>Nature Nanotechnology</em> publication. This experience further emboldened my career aspirations by 1) giving me the freedom to independently conceptualize and carry out a project while managing several failures along the way and 2) allowing me to lay a foundation for lab members to carry the work forward.</p>

<p><img src="/images/AmgenPoster.jpg" style="float:center;width:100%;height:auto;" /></p>

<ul>
  <li>Zhang, H.* Goh, N.S.*, Wang, J., Demirer, G.S., <strong>Butrus, S.</strong>, Park, S-J, Landry, M.P. <ins>Nanoparticle Cellular Internalization is Not Required for RNA Delivery to Mature Plant Leaves</ins>. <em>Nature Nanotechnology</em> (2021) 17, 2: pp. 197-205.</li>
  <li>Wang, J.W., Grandio, E.G., Newkirk, G.M., Demirer, G.S., <strong>Butrus, S.</strong>, Giraldo, J.P., Landry, M.P. <ins>Nanoparticle-mediated genetic engineering of plants</ins>. <em>Molecular Plant</em> (2019) 12; pp. 1037-1040.</li>
  <li><strong>Butrus, S.</strong>, Demirer, G., Goh, N., Zhang, H., Cunningham, F., Landry, M. <ins>Development and Characterization of Gold Nanoparticles for Plant Genetic Engineering </ins>
    <ul>
      <li><strong>Oral</strong>: UC Berkeley Amgen Scholars Program Symposium, Berkeley, CA (2018); ABRCMS Annual Meeting, Indianapolis, IN (2018); GCURS Rice University, Houston, TX (2018)</li>
      <li><strong>Poster</strong>: UC Berkeley Amgen Scholars Program Symposium, Berkeley, CA (2018); BMES Annual Meeting, Atlanta, GA (2018); AIChE Annual Meeting, Pittsburgh, PA (2018)</li>
    </ul>
  </li>
</ul>

<p>This work was performed under the mentorship of <a href="https://landrylab.com/">Prof. Markita Landry</a> at UC Berkeley within the <a href="https://amgenscholars.berkeley.edu/#:~:text=About%20the%20Program,pursuing%20careers%20in%20the%20sciences.">Amgen Scholars Program</a>.</p>

<h3 id="ii-tuning-size-and-charge-of-a-multivalent-polymer-library-for-enhanced-drug-delivery-to-cartilage"><strong>II. Tuning size and charge of a multivalent polymer library for enhanced drug delivery to cartilage</strong></h3>

<p>Efficient transport of therapeutics through dense biological tissues has proven to be a complex challenge, especially for osteoarthritis (OA), a degenerative joint disease characterized by gradual cartilage degradation affecting ~30 million individuals annually. In this work, I focused on addressing the transport of a potential drug delivery vehicle into cartilage for the treatment of OA. The avascular, densely packed, and negatively charged extracellular matrix of cartilage has been a barrier to the transport and, thus, therapeutic effect of even locally administered drugs. I probed whether the uptake-to-cytotoxicity ratio of a nanocarrier in cartilage can be maximized by systematically tuning its physicochemical properties. Taking advantage of the negatively-charged cartilage, I tuned the size and charge of cationic polymers by grafting their periphery with a library of polyethylene glycol molecules. The results revealed a significant influence of size and charge on uptake capabilities of the polymers and their cytotoxicity in bovine cartilage explants and chondrocytes. Furthermore, these findings will inform key design principles of drug delivery vehicles for the potential treatment of OA patients.</p>

<p><img src="/images/EBICS_poster.jpg" style="float:center;width:100%;height:auto;" /></p>

<ul>
  <li><strong>Butrus, S.</strong>, Geiger, B., Grodzinsky, A.J., Hammond, P.T., <ins>Tuning Size and Charge of a Multivalent Polymer Library for Enhanced Drug Delivery to Cartilage</ins>
    <ul>
      <li><strong>Oral</strong>: AIChE Annual Conference, Minneapolis, MN (2017)</li>
      <li><strong>Poster</strong>: MIT Summer Research Programs Annual Poster Session, Cambridge, MA (2017); EBICS Annual Conference, Atlanta, GA (2017); BMES Annual Conference, Phoenix, AZ (2017); NanoDDS Annual Meeting, Ann Arbor, MI (2017)</li>
    </ul>
  </li>
</ul>

<p>This work was performed under the mentorship of <a href="https://scholar.google.com/citations?user=ziQOcJUAAAAJ&amp;hl=en">Dr. Brett Geiger</a> in Prof. Paula Hammond’s <a href="https://hammondlab.mit.edu/">group</a> at MIT as part of the <a href="https://ebics.mit.edu/">NSF Center on EBICS</a>.</p>

<h3 id="iii-nanomechanical-properties-of-zwitterionic-polymer-brushes"><strong>III. Nanomechanical properties of zwitterionic polymer brushes</strong></h3>

<p>I began this project because I was intrigued by interfacial phenomena, namely those governing the performance of polymer brush coatings in supporting exogenous donor-free culture of human embryonic stem cells (hESCs). Prior work had indicated that surface rigidity and elasticity influence cell attachment, spreading, proliferation, and differentiation, but no studies had directly investigated the interaction between thickness, wettability, and nanomechanical properties of the surfaces developed by our group. I probed 1) whether simple reaction parameters that govern thickness and wettability can modulate the nanomechanical properties of these synthetic surfaces and 2) whether hESC propagation responded to these variations in architectural regimes.</p>

<p><img src="/images/synth_scheme_brushes.png" style="float:center;width:100%;height:auto;" /></p>

<p><img src="/images/brush_scheme.png" style="float:left;width:40%;height:auto;" />
<br /><br /></p>

<p><img src="/images/brush_boxplots.png" style="float:right;width:60%;height:auto;" /></p>

<p><br /><br /><br /><br /><br /><br /><br /><br /><br /><br /></p>

<p>Through a statistical design of experiments approach, I employed chemical vapor deposition and atom transfer radical polymerization to design polymer brush coatings of varying thicknesses and wettabilities then acquired their reduced Young’s modulus, $E_r$, and hardness, $H$, via nanoindentation. Analysis of variance for $E_r$ and $H$ suggested that the nanomechanical properties are shaped by a complex interplay between thickness and wettability. This work explained previous results2 from our group, where cells preferred the moderately thick and moderately hydrophobic coatings because they had the optimal $ E_r $ and $H$ for rapid hESC propagation.</p>

<p>This work was performed under the mentorship of <a href="https://kumarbiomaterials.org/">Prof. Ramya Kumar</a> in Prof. Joerg Lahann’s <a href="https://www.lahannlab.com/">group</a> at the University of Michigan.</p>

      </div>
    </main><footer class="site-footer h-card">
  <data class="u-url" href="/"></data>

  <div class="wrapper">


    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          
            </li><li><a class="u-email" href="mailto:salwan [at] berkeley [dot] edu">salwan [at] berkeley [dot] edu</a></li></ul>
      </div>

      <div class="footer-col footer-col-2"><ul class="social-media-list"><li><a href="https://github.com/salwanbutrus"><svg class="svg-icon"><use xlink:href="/assets/minima-social-icons.svg#github"></use></svg> <span class="username">salwanbutrus</span></a></li><li><a href="https://www.twitter.com/salwan_butrus"><svg class="svg-icon"><use xlink:href="/assets/minima-social-icons.svg#twitter"></use></svg> <span class="username">salwan_butrus</span></a></li></ul>
</div>

      <div class="footer-col footer-col-3">
        <p>© 2022 Salwan Butrus</p>
      </div>
    </div>

  </div>

</footer>
</body>
  
  <script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [['$','$'], ['\\(','\\)']],
      processEscapes: true
    }
  });
  </script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>

</html>
